app/views/eu_exit_business_readiness_requests/_request_details.html.erb

```diff
@@ -3,32 +3,17 @@
 </p>
 
 <p>
-  The <a href="https://www.gov.uk/business-uk-leaving-eu">EU Exit business
-  readiness</a> tool aims to help businesses find content that gives them
-  actionable steps to prepare for EU Exit. It asks users a series of questions
-  before showing them links to relevant content in a
-  <a href="https://www.gov.uk/find-eu-exit-guidance-business">finder</a>.
-  Content shown in the finder is already published on GOV.UK.
+  Content tagged to the <a href="https://www.gov.uk/business-uk-leaving-eu">Brexit business
```

Comment:
The Google doc has changed the URL from `https://www.gov.uk/business-uk-leaving-eu` to `https://www.gov.uk/find-eu-exit-guidance-business`.

Reply:
Good spot! Updating now...

Reply:
Updated and fixed up.